src/React.re

```diff
@@ -319,9 +391,16 @@ external useMemo7:
   ([@bs.uncurry] (unit => 'any), ('a, 'b, 'c, 'd, 'e, 'f, 'g)) => 'any =
   "useMemo";
 
-/* This is used as return values  */
+[@text "{2 Callback}"];
+
+/** This type forces BuckleScript to treat the hook return value as a
+    function. */
 type callback('input, 'output) = 'input => 'output;
 
+/* TODO: should we even bother to bind the first two? The whole point of
```

Comment:
@rickyvetter (and/or others): does it make sense to use `useCallback` and `useMemo` hooks without dependencies?